packages/nc-gui/components/general/FullScreen.vue

```diff
@@ -0,0 +1,29 @@
+<script setup lang="ts">
+import { useSidebar } from '#imports'
+
+const rightSidebar = useSidebar('nc-right-sidebar')
+const leftSidebar = useSidebar('nc-left-sidebar')
+
+const isFullScreen = computed({
+  get: () => !(rightSidebar.isOpen.value || leftSidebar.isOpen.value),
+  set: (value) => {
+    rightSidebar.toggle(!value)
+    leftSidebar.toggle(!value)
+  },
```

Comment:
Isn't this a double negation?
When clicking the div below you set `isFullScreen = !isFullScreen`
and then you take that value and negate it again by doing `!value`